autogtp/Game.cpp

```diff
@@ -23,11 +23,12 @@
 #include <QFileInfo>
 #include "Game.h"
 
-Game::Game(const QString& weights, const QString& opt, const QString& binary) :
+Game::Game(const QString& weights, const QString& opt, const QString& binary,
```

Comment:
Game is used from autogtp too. So I guess this change breaks autogtp.

Reply:
I'll double check but it is defaulted in the constructor in the Game.h

Reply:
Yep, still works for me.